ts/features/bonus/bpd/saga/orchestration/onboarding/enrollToBpd.ts

```diff
@@ -34,6 +35,7 @@ function* enrollToBpdWorker() {
   );
 
   if (enrollResult.payload.enabled) {
+    yield put(bpdAllData.request());
```

Comment:
I guess this request has a side effect:
`bpdAllData` requests also for the citizen activation status. The response (since the citizen is enrolled) invalidates the onboarding state (it becomes false due to [this](https://github.com/pagopa/io-app/blob/a6c2cfb2853e04b21be3c8c4fe8b1ad22f464bc9/ts/features/bonus/bpd/store/reducers/onboarding/ongoing.ts#L28) )

I noted mainly this effect

| before  | now |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/103408675-c8582f00-4b63-11eb-8f84-3acfcdf872fa.png) | ![now](https://user-images.githubusercontent.com/822471/103408689-d1e19700-4b63-11eb-964d-90e715ed007e.png)

Reply:
tnx for the catch, solved in d4ede1e, removing the condition (legacy, not needed anymore)